resmio_conversion.js

```diff
@@ -0,0 +1,195 @@
+/*!
+ * ===========================================
+ * General Conversion Tracking v1.0
+ * Copyright 2017 resmio GmbH (Philipp Sahner)
+ * ===========================================
+ */
++function ($) {
+  /*
+   * +++++++++++++++++++++++++++++++++++++++++++++++++++++
+   * Check the url for parameters when document has loaded
+   * and handle it accordingly
+   * +++++++++++++++++++++++++++++++++++++++++++++++++++++
+   */
+  $(document).ready(function(){
+    // get the url, check it for parameters and update the links
+ 	  var url = window.location.href,
+        obj = getAllUrlParams(url);
+        isGclid = false;
+        isObj = false;
```

Comment:
instead of this you can assign it directly here 
`isObject = (!($.isEmptyObject(obj)))`